### PR TITLE
Some code cleanup

### PR DESCRIPTION
**Pull request by mathieucarbou**

### CODE REVIEW

**Review by vortigont — COMMENTED** [review]
Nice tidy-up in code, @mathieucarbou! :)

File: src/literals.h [hunk 1]
```diff
@@ -5,46 +5,79 @@ namespace asyncsrv {
 static constexpr const char* empty = "";
 
 #ifndef ESP8622
```
Comment (Pablo2048) [inline src/literals.h:7]: Isn't here a typo? (ESP8266 instead of ESP8266)

**Comment by mathieucarbou** [comment]
> Nice tidy-up in code, @mathieucarbou! :)

Did you see also the perf improvements ? Finally get to fix a nasty bug in AsyncTCP, and made the lib compatible with AsyncTCPSock, which has better perf:

https://github.com/mathieucarbou/ESPAsyncWebServer?tab=readme-ov-file#performance

File: src/literals.h (hunk 1 above)
Comment (mathieucarbou) [reply src/literals.h:7]: OMG! @vortigont : this bug is there since a while! That might be why ESP8266 users have some memory issues ?

Comment (vortigont) [reply src/literals.h:7]: Wow! Nice catch! :)

**Comment by vortigont** [comment]
> Did you see also the perf improvements ?

Need to take a closer look into this, kind of overloaded lately with main job :( not that much time for coding

You've been doing a hell of a progress, Mathieu!

**Pull request merged**